test/integration/bare-imports/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "src"
```

Comment:
Can you add support for custom `"paths"` here as well? That's the main feature request that I've seen since we added our original support for Top-Level Imports.

Reply:
True. Good catch. My original PR was for `baseUrl` only and then I forgot to update the tests :)

Reply:
Done